platform/security/sts/security-sts-crlinterceptor/src/main/java/org/codice/ddf/security/sts/crl/CRLInterceptor.java

```diff
@@ -31,31 +24,24 @@
 import org.apache.cxf.phase.AbstractPhaseInterceptor;
 import org.apache.cxf.phase.Phase;
 import org.apache.cxf.transport.http.AbstractHTTPDestination;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-import ddf.security.common.audit.SecurityLogger;
+import org.codice.ddf.security.handler.pki.CrlChecker;
 
 /**
  * Interceptor that checks an incoming message against a defined certification
  * revocation list (CRL).
- *
  */
 public class CRLInterceptor extends AbstractPhaseInterceptor<Message> {
 
-    private static final Logger LOGGER = LoggerFactory.getLogger(CRLInterceptor.class);
-
-    private boolean isEnabled = false;
-
-    private CRL crl;
+    protected CrlChecker crlChecker;
```

Comment:
Is this `protected` only for unit testing? If so, we should either make it package private and add a comment or make it `private` and add a setter method or pass it in the constructor.